### PR TITLE
Memory metering

### PR DESCRIPTION
**Pull request by turbolent**
Closes dapperlabs/cadence-private-issues#36

Huge shout-out to @SupunS @dsainati1 @robert-e-davidson3 👏 👏 👏 

---

- [x] Targeted PR against `master` branch
- [x] Linked to Github issue with discussion and accepted design OR link to spec that describes this work
- [x] Code follows the [standards mentioned here](https://github.com/onflow/cadence/blob/master/CONTRIBUTING.md#styleguides)
- [x] Updated relevant documentation 
- [x] Re-reviewed `Files changed` in the Github PR explorer
- [x] Added appropriate labels

### CODE REVIEW

File: docs/language/accounts.mdx [hunk 1]
```diff
@@ -88,6 +88,19 @@ to the `prepare` phase of the transaction.
 
       let keys: AuthAccount.Keys
 
+      // Key management
```
Comment (dsainati1) [inline docs/language/accounts.mdx:91]: Is this supposed to include the public key API changes?

File: runtime/interpreter/conversion.go [hunk 2]
```diff
@@ -85,6 +85,7 @@ func ByteValueToByte(element Value) (byte, error) {
 	return b, nil
 }
 
+// TODO: meter
```
Comment (SupunS) [inline runtime/interpreter/conversion.go:88]: should be safe to remove this todo?

Comment (turbolent) [reply runtime/interpreter/conversion.go:88]: There seem to be still some outstanding TODOs and PRs too:
- https://github.com/onflow/cadence/pull/1578
- https://github.com/onflow/cadence/pull/1598

File: docs/language/accounts.mdx (hunk 1 above)
Comment (turbolent) [reply docs/language/accounts.mdx:91]: This PR just reverts the removal of the deprecated functions. This documentation should have been added back to master already before, but it seems like it wasn't

**Review by SupunS — APPROVED** [review]
🎉  🎉  🎉

**Pull request merged**